provision/labels.go

```diff
@@ -37,6 +37,7 @@ const (
 	LabelAppVersion   = "app-version"
 	LabelAppTeamOwner = "app-team"
 
+	LabelIsTsuruJob   = "is-tsuru-job"
```

Comment:
Why not just `is-job`?

Reply:
Guess you're right... Lazy thinking on my part, I just thought we could kill two labels with one using this, but makes no sense since I wasn't brave enough to remove `is-tsuru` label 😜